app/uk/gov/hmrc/economiccrimelevyaccount/utils/CorrelationIdHelper.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyaccount.utils
+
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.http.HeaderCarrierConverter
+import java.util.UUID
+import play.api.mvc.Request
+
+object CorrelationIdHelper {
```

Comment:
This needs some unit tests.

See following example:

[https://github.com/hmrc/ers-submissions/blob/main/test/utils/CorrelationIdHelperSpec.scala](https://github.com/hmrc/ers-submissions/blob/main/test/utils/CorrelationIdHelperSpec.scala)